README.md

```diff
@@ -93,6 +93,148 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
 
 ![ screencast ](images/0.3.0/reg-stack-analysis.gif)
 
+## Using the CRDA Ignore Vulnerability Feature
+
+If a user wishes to ignore all the vulnerabilities, or a specific set of security vulnerabilities of a particular dependency, or set of dependencies, it can be done by adding "crdaignore" followed by an optional list consisting of the security vulnerabilities to ignore as comments in the manifest file for that particular dependency for the maven, pypi and golang ecosystems, or by adding a JSON "crdaignore" object to an npm manifest file as shown in the examples below.
```

Comment:
If users wish to ignore vulnerabilities for a dependency, it can be done by adding "crdaignore" as a comment in manifest file for Python, Maven, Golang and as a JSON for Node ecosystem because Node manifest files dont support comments hence it has to be given inside a JSON.
If "crdaignore" is followed by a list of comma separated Snyk vulnerability IDs then only listed vulnerabilities will be ignored during analysis, in case "crdaignore" is not followed by any list all vulnerabilities present in package will be ignored.

# Examples 
## Python 
Ignore all vulnerabilities in fastapi and few for flask

fastapi==0.36.0 #crdaignore
sceptre==2.2.1
flask==1.0 #crdaignore [<Snyk vulnerability ID 1 >, <Snyk vulnerability ID 2 >]

## Maven
followed by other ecosystems